tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -101,6 +101,16 @@ def wait_for_app_update_to_clear(self):
         Wait(self.marionette).until(lambda m: element.location['y'] == (0 - element.size['height']))
 
     @property
+    def screen_width(self):
```

Comment:
I guess this could be called height as well, since it's the height of the system app, but I guess it doesn't matter.